draft-ietf-tls-tls13.md

```diff
@@ -2093,11 +2093,13 @@ As of TLS 1.3, all clients and servers will send at least
 one extension (at least "key_share" or "pre_shared_key").
 
 server_version
-: This field will contain the lower of that suggested by the client
-  in the ClientHello and the highest supported by the server.
-  Servers MUST accept ClientHello versions higher than those supported.
-  For this version of the specification, the version is { 3, 4 }.
-  (See {{backward-compatibility}} for details about backward compatibility.)
+: This field contains the version of TLS negotiated for this session.  Servers
+  MUST select the lower of the highest supported version and the version
```

Comment:
"highest supported version" is too ambiguous; "highest supported server version" would be preferable.

The whole sentence could be a little more clear, however. I suggest something like:

"Servers MUST select the lower of the two endpoints' supported versions: either the highest supported server version or the version offered by the client in its ClientHello."